pkg/globalconfig/global_config.go

```diff
@@ -36,33 +37,35 @@ type ProjectInfo struct {
 
 // GlobalConfig is the struct defining ddev's global config
 type GlobalConfig struct {
-	OmitContainersGlobal         []string                `yaml:"omit_containers,flow"`
-	NFSMountEnabledGlobal        bool                    `yaml:"nfs_mount_enabled"`
-	MutagenEnabledGlobal         bool                    `yaml:"mutagen_enabled"`
-	InstrumentationOptIn         bool                    `yaml:"instrumentation_opt_in"`
-	RouterBindAllInterfaces      bool                    `yaml:"router_bind_all_interfaces"`
-	InternetDetectionTimeout     int64                   `yaml:"internet_detection_timeout_ms"`
-	DeveloperMode                bool                    `yaml:"developer_mode,omitempty"`
-	InstrumentationUser          string                  `yaml:"instrumentation_user,omitempty"`
-	LastStartedVersion           string                  `yaml:"last_started_version"`
-	UseHardenedImages            bool                    `yaml:"use_hardened_images"`
-	UseLetsEncrypt               bool                    `yaml:"use_letsencrypt"`
-	LetsEncryptEmail             string                  `yaml:"letsencrypt_email"`
-	AutoRestartContainers        bool                    `yaml:"auto_restart_containers"`
-	FailOnHookFailGlobal         bool                    `yaml:"fail_on_hook_fail"`
-	WebEnvironment               []string                `yaml:"web_environment"`
-	DisableHTTP2                 bool                    `yaml:"disable_http2"`
-	TableStyle                   string                  `yaml:"table_style"`
-	SimpleFormatting             bool                    `yaml:"simple_formatting"`
-	RequiredDockerComposeVersion string                  `yaml:"required_docker_compose_version,omitempty"`
-	UseDockerComposeFromPath     bool                    `yaml:"use_docker_compose_from_path,omitempty"`
-	MkcertCARoot                 string                  `yaml:"mkcert_caroot"`
-	ProjectTldGlobal             string                  `yaml:"project_tld"`
-	XdebugIDELocation            string                  `yaml:"xdebug_ide_location"`
-	NoBindMounts                 bool                    `yaml:"no_bind_mounts"`
-	UseTraefik                   bool                    `yaml:"use_traefik"`
-	WSL2NoWindowsHostsMgt        bool                    `yaml:"wsl2_no_windows_hosts_mgt"`
-	ProjectList                  map[string]*ProjectInfo `yaml:"project_info"`
+	OmitContainersGlobal             []string                `yaml:"omit_containers,flow"`
```

Comment:
In general, please avoid letting your IDE make significant changes that are not related to the PR.

Reply:
I've added 2 new configs which caused the new indention, related to this PR though.

Reply:
Oh, I see, the new configs were longer, and the IDE decided to adjust everything as a result.

Reply:
Exactly :)